values-hr

```diff
@@ -0,0 +1,181 @@
+        <!--HINTS-->
+            <string name="slide_arch_hint">Aplikacija automatski određuje arhitekturu vašeg uređaja</string>
+            <string name="slide_android_hint">Aplikacija automatski određuje verziju Androida na vašem uređaju</string>
+            <string name="slide_variant_hint">Aplikacija automatski određuje varijantu instaliranu na vašem uređaju</string>
+        <!--VARIOUS-->
+            <string name="detected">otkriveno</string>
+
+    <!--______PREFERENCES______-->
+        <!--LABELS-->
+            <string name="pref_header_interface">Sučelje</string>
+            <string name="pref_header_install">Instalacija</string> <!--noun-->
+            <string name="pref_header_download">Preuzimanje</string> <!--noun-->
+            <string name="label_dark_mode">Tamna tema</string>
+            <string name="new_folder">Nova mapa</string>
+            <string name="pref_header_gapps_selection">Odabir GApps-a</string>
+            <string name="action_settings">Postavke</string>
+            <string name="label_wifi_only">Samo Wi-Fi</string>
+            <string name="label_download_dir">Mapa preuzimanja</string>
+            <string name="label_wipe_cache">Izbriši cache/dalvik</string>
+            <string name="label_download_md5">Preuzmi MD5</string>
+            <string name="label_automatic_cleanup">Automatski izbriši preuzimanja</string>
+            <string name="label_number_packages_kept">Broj paketa za zadravanje prije brisanja</string>
+            <string name="label_download_versionlog">Preuzmi zapis verzije</string>
+            <string name="label_root_mode">Upotrebljavaj root dopuštenje</string>
+            <string name="label_pref_install_warning">Prikaži upozorenje za predinstalaciju</string>
+            <string name="label_pref_delete_warning">Prikaži upozorenje za predbrisanje</string>
+        <!--SUMMARIES-->
+            <string name="explanation_dark_mode">Tamna tema poboljšava čitljivost pri uvjetima slabog osvjetljenja</string>
+            <string name="summary_wifi_only">Download over Wi-Fi only</string>
+            <string name="explanation_newest_version">Delete outdated version if a new similar variant has been downloaded</string>
+            <string name="explanation_keep_packages">Stariji slični paketi koji će se zadržati: %1$d</string>
```

Comment:
Would you be able to rewrite this one in a manner that would make %1$d (= a number) as part of the sentence? It would look more nice (it is shown as a sub-text in the settings)